feconf.py

```diff
@@ -78,7 +78,7 @@
 # A mapping of interaction ids to classifier properties.
 INTERACTION_CLASSIFIER_MAPPING = {
     'TextInput': {
-        'algorithm_id': 'LDAStringClassifier',
+        'algorithm_id': 'TextClassifier',
```

Comment:
Why wasn't this modified before @anmolshkl (when you implemented TextClassifier on Oppia side)

Also have you done the end to end tests (manually) of running TextClassifier i.e. start Oppia-ml and Oppia, change all the constants and enable classification framework, create or load any exploration that uses text classifier, and then check logs of Oppia-ml to see if classifier is trained, check logs of Oppia and see if trained model has been stored, also playtest exploration once the trained model has been stored and see if it predicts correct values?